oap-server/server-starter/src/main/resources/ui-initialized-templates/os_linux/linux-service.json

```diff
@@ -156,10 +156,12 @@
             "showYAxis": true
           },
           "metrics": [
-            "meter_vm_filesystem_percentage"
+            "meter_vm_filesystem_percentage",
+            "meter_vm_filesystem_percentage_telegraf"
           ],
           "metricTypes": [
-            "readLabeledMetricsValues"
+            "readLabeledMetricsValues",
+            "readMetricsValues"
```

Comment:
@Fine0830 I noticed this kind of hybrid setting, one for labeled metrics, the other for a single value. Is this supported in the plan?

@soander Please post screenshot about how it looks like in OpenTelemetry and Telegraf cases.

Reply:
I don't think these two metrics will appear at the same time in the same dashboard. The two indicators are only for compatibility with the data that the chart can query.